app/config/team_members.yml

```diff
@@ -26,9 +26,10 @@ parameters:
             active: true
         carusogabriel:
             name: 'Gabriel Caruso'
+            twitter: carusogabriel
             github: carusogabriel
             avatarUrl: 'https://avatars3.githubusercontent.com/u/16328050?v=4'
-            website: 'https://twitter.com/carusogabriel'
+            website: ''
```

Comment:
`~`?

Reply:
Empty string is used everywhere else so I used the same. @jwage any reason for `''`?

Reply:
That was just from the github import. Null is fine.